implementations.md

```diff
@@ -131,6 +131,8 @@ Schema generation
     -   <a href="http://schemaguru.snowplowanalytics.com/" id="link-impl-guru-ui">Schema Guru Web UI</a> - derives precise Schemas using several JSON instances. Based on [Schema Guru](link-impl-guru)
 -   Visual Studio
     -   <a href="http://visualstudiogallery.msdn.microsoft.com/b4515ef8-a518-41ca-b48c-bb1fd4e6faf7" id="link-impl-vs">JSON Schema Generator</a> - free extension
+-   UML
```

Comment:
Based on the other entries here (ex. Visual Studio), I think "Sparx Enterprise Architect" may be a better heading.